rfcs/0001-reason-message-diff.md

```diff
@@ -0,0 +1,53 @@
+### Context:
+
+When kpack produces new builds today, a Reason is given which may be one of five values: COMMIT, TRIGGER, CONFIG, STACK, BUILDPACK
```

Comment:
It is worth noting that a build may be multiple of those values. As in a build could have `COMMIT,CONFIG`. In extreme cases it may even have them all `COMMIT,CONFIG,STACK,BUILDPACK`

Reply:
Yes definitely, i tried to reference that in the complexity section